akka-docs/src/main/paradox/typed/coexisting.md

```diff
@@ -116,13 +116,16 @@ Scala
 Java
 :  @@snip [TypedWatchingClassicTest.java](/akka-actor-typed-tests/src/test/java/jdocs/akka/typed/coexistence/TypedWatchingClassicTest.java) { #typed }
 
+Note that when sending from a typed actor to a classic @scala[`ActorRef`]@java[`ActorRef`] there is no sender in scope as in classic.
+The typed sender should use its own @scala[`ActorContext[T].self`]@java[`ActorContext[T].getSelf()`] explicitly, as shown in the snippet.
+
```

Comment:
this can be within `@@@div { .group-scala }` because it's only relevant to Scala

and then the `@scala` and `@java` can be removed